packages/gatsby-remark-code-repls/src/index.js

```diff
@@ -114,17 +126,22 @@ module.exports = (
                   }
                   return map
                 }, {}),
+                main: filesPaths[0].fileName,
               },
             },
-            "index.js": {
-              content: code,
-            },
             "index.html": {
               content: html,
             },
           },
         }
 
+        filesPaths.forEach((path, i) => {
+          const code = fs.readFileSync(path.filePath, `utf8`)
+          parameters.files[path.fileName] = {
```

Comment:
I think we should actually use the full path, not just file name. Might be conflicts otherwise.